fedimint-core/src/config.rs

```diff
@@ -747,11 +747,11 @@ pub trait TypedServerModuleConfig: DeserializeOwned + Serialize {
 }
 
 /// Things that a `distributed_gen` config can send between peers
-// TODO: Needs to be modularized in case modules want to send new message types for DKG
 #[derive(Serialize, Deserialize, Debug, Clone)]
 pub enum DkgPeerMsg {
     PublicKey(secp256k1::PublicKey),
     DistributedGen(SupportedDkgMessage),
+    Module(Vec<u8>),
```

Comment:
Eventually I envision this to be used for all module DKGs, even the ones that have a more specific message type right now. If we properly abstract the DKGs for each key type we should be able to make these messages more about the usage/instance of the key than its type.

That would also mean making it a proper `Dyn*` type eventually, but we got more important things to do right now, this unblocks you and we don't give stability guarantees for DKG for now as it's only run on startup and we can assume everyone to be on the same version.

Reply:
Yep, sounds good to me.